scalr/data_source_scalr_module_version.go

```diff
@@ -46,7 +46,25 @@ func dataSourceModuleVersionRead(ctx context.Context, d *schema.ResourceData, me
 	var version string
 	if v, ok := d.GetOk("version"); ok {
 		version = v.(string)
-		mv, err = scalrClient.ModuleVersions.ReadBySemanticVersion(ctx, module.ID, version)
+		if !scalr.ValidStringID(module.ID) {
```

Comment:
Add this to `version` `Schema` by using `ValidateFunc`.

Reply:
moved the validation to ValidateFunc